Documentation/source/Migration.md

```diff
@@ -0,0 +1,202 @@
+```
+#### Certificate pinning
+
+If you're using _Certificate pinning_ and have the following lines in your code base:
+```swift
+let viewController = GiniVision.viewController(withClient: client,
+                                               configuration: giniConfiguration,
+                                               resultsDelegate: resultsDelegate,
+                                               publicKeyPinningConfig: yourPublicPinningConfig,
+                                               documentMetadata: documentMetadata,
+                                               api: .accounting)
+
+present(viewController, animated: true, completion:nil)
+```
+
+ It should now become:
+```swift
+let viewController = GiniPayBank.viewController(withClient: client,
+                                               configuration: giniPayBankConfiguration,
+                                               resultsDelegate: giniCaptureResultsDelegate,
+                                               publicKeyPinningConfig: yourPublicPinningConfig,
+                                               documentMetadata: documentMetadata,
+                                               api: .default)
+
+present(viewController, animated: true, completion:nil)
+```
+
+#### Only UI
+
+For using only the UI and the handling the analysis processes either was used Gini API SDK or  your own implementation of the [Gini API](https://developer.gini.net/gini-api/html/index.html). The `UIViewController` initialization was following:
+
+```swift
+let viewController = GiniVision.viewController(withDelegate: self,
+                                               withConfiguration: giniConfiguration)
+
+present(viewController, animated: true, completion: nil)
+```
+
+Now to handle all the analysis processes you should use the new [Gini Pay Api Library](https://github.com/gini/gini-pay-api-lib-ios) or your own implementation of the new [Gini Pay API](https://pay-api.gini.net/documentation/#gini-pay-api-documentation-v1-0) and just get the `UIViewController` as follows:
+
+```swift
+let viewController = GiniPayBank.viewController(withDelegate: self,
+                                               withConfiguration: giniPayBankConfiguration)
+
+present(viewController, animated: true, completion: nil)
+```
+
+## Component API
+
+Previously setting the configuration `GiniConfiguration` had been looked like:
+```swift
+let viewController = GiniVision.viewController(withDelegate: self,
+                                               withConfiguration: giniConfiguration)
+
+present(viewController, animated: true, completion: nil)
+```
+
+Now alternately of using `GiniConfiguration` in the Gini Pay Bank SDK was introduced `GiniPayBankConfiguration`.
+The configuration for `GiniCapture` should be set explicitly as it's shown below:
+
+```swift
+let giniPayBankConfiguration = GiniPayBankConfiguration()
+.
+.
+.
+GiniCapture.setConfiguration(giniPayBankConfiguration.captureConfiguration())
```

Comment:
As a reader I am now wondering how do I get the `UIViewController` after setting the configuration?